test/CMakeLists.txt

```diff
@@ -86,12 +87,13 @@ if(EXISTS ${TEST_NC_FILE_SPHERICAL})
     message(STATUS "Found Spherical NC File: ${TEST_NC_FILE_SPHERICAL}") 
     set(TEST_NC_FILE ${TEST_NC_FILE_SPHERICAL})
     # this test only works for spherical NC files!
-    pmpo_add_test(testFortranMPAdvection ./testFortranMPAdvection ${TEST_NC_FILE}) 
+    pmpo_add_test(testFortranMPAdvection 1 ./testFortranMPAdvection ${TEST_NC_FILE}) 
 else()
     set(TEST_NC_FILE ${TEST_NC_FILE_PLANAR})
 endif()
-pmpo_add_test(testFortranReadMPAS ./testFortranReadMPAS ${TEST_NC_FILE})
-pmpo_add_test(testFortranCreateRebuildMPs ./testFortranCreateRebuildMPs ${TEST_NC_FILE})
+pmpo_add_test(testFortranReadMPAS 1 ./testFortranReadMPAS ${TEST_NC_FILE})
+pmpo_add_test(testFortranCreateRebuildMPs 1 ./testFortranCreateRebuildMPs ${TEST_NC_FILE})
 
-pmpo_add_test(test_timing ./timeAssmblyWachspress 1 ${TEST_NC_FILE})
-pmpo_add_test(test_wachspress ./testWachspress ${TEST_NC_FILE})
+pmpo_add_test(test_timing 1 ./timeAssmblyWachspress 1 ${TEST_NC_FILE})
+pmpo_add_test(test_wachspress 1 ./testWachspress ${TEST_NC_FILE})
+pmpo_add_test(test_migration 8 ./testMigration ${TEST_NC_FILE}) 
```

Comment:
If MPI isn't required in PolyMPO, we should wrap the call to `pmpo_add_test` with something like
`if(MPIRUN_EXECUTABLE)`.